lib/hasBlock.js

```diff
@@ -0,0 +1,7 @@
+function hasBlock(rule) {
```

Comment:
I added it to the lib folder as it was next to the other files "hasDecls.js" and "hasRules.js". We could move them all to a /util folder?